projects/dsp-ui/src/lib/viewer/views/list-view/resource-grid/resource-grid.component.html

```diff
@@ -10,13 +10,13 @@
             <mat-card-title class="res-class-value">{{ res.label }}</mat-card-title>
         </mat-card-header>
         <mat-card-content class="grid-card-content" *ngFor="let prop of res.properties | keyvalue">
-            <div *ngFor="let val of prop.value | keyvalue">
+            <div *ngFor="let val of prop.value">
```

Comment:
@flavens I removed the second `keyvalue` pipe because of a compiler problem. I think it is not necessary since `prop.value` is a `ReadValue[]` and thus iterable.